plugins/modules/packaging/os/pacman_key.py

```diff
@@ -322,7 +331,6 @@ def main():
         ),
         supports_check_mode=True,
         mutually_exclusive=(('data', 'file', 'url', 'keyserver'),),
-        required_one_of=[('data', 'file', 'url', 'keyserver'),],
```

Comment:
Instead of removing this (and adding manual checks), better add
```.py
required_if=[('state', 'present', ('data', 'file', 'url', 'keyserver'), True)],
```
([Docs](https://docs.ansible.com/ansible/latest/dev_guide/developing_program_flow_modules.html#dependencies-between-module-options))